stan/math/prim/mat.hpp

```diff
@@ -6,6 +6,8 @@
 #include <stan/math/prim/arr/meta/is_vector.hpp>
 #include <stan/math/prim/arr/meta/length.hpp>
 
+#include <stan/math/prim/mat/fun/Eigen.hpp>
```

Comment:
Our includes are a bit confusing - there are many things that use Eigen types between where I moved this include and where it used to live further down... but this seems to work